csrc/ops/arith.cpp

```diff
@@ -2299,11 +2299,14 @@ namespace {
 //   mat2   [g, k, n]
 //   offset [g]
 //   output -> [m, n, [rk]]
-TensorView* createGroupedMmaOutput(
+ScaledTensorView createGroupedMmaOutput(
     TensorView* mat1,
     TensorView* mat2,
     TensorView* offsets,
-    std::optional<DataType> dtype) {
+    std::optional<DataType> dtype,
```

Comment:
```suggestion
    DataType dtype,
```
and `DataType::Null` later